src/contribute/eclipse.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: section
+title: "Beam Eclipse Tips"
+permalink: /contribute/eclipse/
+section_menu: section-menu/contribute.html
+---
+
+# Eclipse
+
+If Eclipse is your preferred development environment, this section may help you
+prepare it to develop Beam code.
+
+Use a recent Eclipse version that includes m2e. Currently we recommend Eclipse
+Neon. Start Eclipse with a fresh workspace in a separate directory from your
```

Comment:
a tangent unrelated to the text - is Neon still the recommended version, or is it Oxygen now?